evals/datasets_/truthfulqa.py

```diff
@@ -9,44 +9,63 @@
 import os
 
 from datasets import Dataset, load_dataset
-from models import translate_google, google_supported_languages
+from models import translate_google, get_google_supported_languages
 
 from datasets_.util import _get_dataset_config_names, _load_dataset
 
 slug_uhura_truthfulqa = "masakhane/uhura-truthfulqa"
+slug_truthfulqa_autotranslated = "fair-forward/truthfulqa-autotranslated"
+
 tags_uhura_truthfulqa = {
     standardize_tag(a.split("_")[0], macro=True): a for a in _get_dataset_config_names(slug_uhura_truthfulqa)
     if a.endswith("multiple_choice")
 }
 
+# Get available auto-translated languages
+try:
+    tags_truthfulqa_autotranslated = {
+        standardize_tag(a, macro=True): a for a in _get_dataset_config_names(slug_truthfulqa_autotranslated)
+    }
+except Exception:
+    tags_truthfulqa_autotranslated = {}
+
 
 def add_choices(row):
     row["choices"] = row["mc1_targets"]["choices"]
     row["labels"] = row["mc1_targets"]["labels"]
     return row
 
 
-def load_truthfulqa(language_bcp_47, nr):
+async def load_truthfulqa(language_bcp_47, nr):
     if language_bcp_47 in tags_uhura_truthfulqa.keys():
-        ds = _load_dataset(slug_uhura_truthfulqa, tags_uhura_truthfulqa[language_bcp_47])
+        ds = _load_dataset(
+            slug_uhura_truthfulqa, tags_uhura_truthfulqa[language_bcp_47]
+        )
         ds = ds.map(add_choices)
-        examples = ds["train"]
         task = ds["test"][nr]
-        return "masakhane/uhura-truthfulqa", examples, task
+        return "masakhane/uhura-truthfulqa", task, "human"
+    elif language_bcp_47 in tags_truthfulqa_autotranslated.keys():
+        # Load from auto-translated dataset (same samples as translation)
+        ds = _load_dataset(slug_truthfulqa_autotranslated, language_bcp_47)
+        test_split = ds["test"] if "test" in ds else ds
+        task = test_split[nr]
+        return slug_truthfulqa_autotranslated, task, "machine"
+    # TODO: add Okapi, TruthfulQA-X @Jonas
     else:
         return None, None, None
 
-
-
 def translate_truthfulqa(languages):
     human_translated = [*tags_uhura_truthfulqa.keys()]
     untranslated = [
         lang
-        for lang in languages["bcp_47"].values[:100]
-        if lang not in human_translated and lang in google_supported_languages
+        for lang in languages["bcp_47"].values[:150]
+        if lang not in human_translated and lang in get_google_supported_languages()
     ]
-    n_samples = 10
+    n_samples = 20
 
+    # Set fixed seed for consistent sample selection across all languages
+    random.seed(42)
```

Comment:
random seed does not seem to be used for anything (there are no random ops anyway)
```suggestion
```